### ISSUE TITLE
Button and Link titles for tooltip on hover

### ISSUE THREAD

**Issue by changecourse** [mozilla-lockbox/lockbox-extension#606]
Do we need to conduct an audit and create button/link titles for additional context for clickable elements throughout Lockbox?

@hmcgaw @m8ttyB and @kimberlythegeek looking for guidance here. 

<img width="282" alt="screen shot 2018-03-02 at 10 38 54 am" src="https://user-images.githubusercontent.com/6164801/36913062-eba4dec4-1e05-11e8-8934-77fc7e53479c.png">

**Comment by kimberlythegeek** [comment]
@changecourse Yes, we do. This will probably link back into my screen reader testing. 

There are several elements that VoiceOver does not read out the name/label of. I'm not sure if this a real issue with lockbox, or if I'm somehow using the screenreader incorrectly.

**Comment by m8ttyB** [comment]
Just a note, there is a chance that we'll (PI) run a full screen reader audit during the course of the next sprint.  In general tool tips (that appear during a mouse hover) are troublesome, some general rules:
* make the tool tip text part of the label for the field or use aria attributes to make the proper logical connections
* ensure that they can be triggered via on-focus and touch events.

**Comment by kimberlythegeek** [comment]
@m8ttyB @changecourse yes. For clickable elements that do not already have a label, adding an `aria-label` and `role="button"` should cover most accessibility requirements.
